package.json

```diff
@@ -36,23 +36,23 @@
         "trim": "ts-node scripts/trim.ts"
     },
     "dependencies": {
-        "@types/node": "^16.11.1",
         "@zowe/cli": "6.36.0",
         "@zowe/imperative": "4.17.0",
-        "husky": "^7.0.2",
```

Comment:
We kind of want this because we require DCO signoff

Reply:
It's included as a dev dep

Reply:
See above